src/model/queries/get_data.js

```diff
@@ -1,5 +1,38 @@
 const dbConnection = require('../database/db_connection');
 
+
+const getSearch = name => new Promise((resolve, reject) => {
+  const sql = {
+    text: 'SELECT * FROM business WHERE name LIKE $1',
+    values: [`%${name}%`],
+  };
+  dbConnection.query(sql, (error, res) => {
+    if (error) return reject(error);
+    resolve(res.rows);
+  });
+});
+
+const getSearchResult = name => new Promise((resolve, reject) => {
+  const sql = {
+    text: 'SELECT * FROM business WHERE name =$1',
+    values: [name],
+  };
+  dbConnection.query(sql, (error, res) => {
+    if (error) return reject(error);
+    resolve(res.rows);
+  });
+});
+
+const getReviews = () => new Promise((resolve, reject) => {
+  const sql = {
+    text: 'SELECT * FROM review ORDER BY id desc',
```

Comment:
Your queries should be separated by concerns rather than functionality.
all queries related to ```business``` should be grouped in the same file, all queries related to ```users``` should be grouped in the same files.

If something goes wrong with the users file, it should not affect the business, and vice versa.